truffle/test/04_workload.js

```diff
@@ -12,22 +12,20 @@ describe('Workload', async () => {
   const NEW_ACCOUNT_PASS = "password";
   
   it("should create an account for testing purposes, not asserting", async () => {
-    NEW_ACCOUNT_ADDR = await web3.personal.newAccount(NEW_ACCOUNT_PASS);
-    await web3.personal.unlockAccount(NEW_ACCOUNT_ADDR, NEW_ACCOUNT_PASS, 1000);
+    NEW_ACCOUNT_ADDR = await web3.eth.personal.newAccount(NEW_ACCOUNT_PASS);
+    await web3.eth.personal.unlockAccount(NEW_ACCOUNT_ADDR, NEW_ACCOUNT_PASS, 1000);
 
-    const txReceiptId = await web3.eth.sendTransaction({
+    await web3.eth.sendTransaction({
```

Comment:
Did you check if this new version already fetch receipt automatically? If so, we probably should validate the status in order to not derive the assertion the following test causing a more than likely confusion on our future debugging,

Reply:
yep it does, ok i will add a check as well for the status,